packages/google-pay-integration/src/gateways/google-pay-gateway.ts

```diff
@@ -32,6 +40,7 @@ export default class GooglePayGateway {
     private _getPaymentMethodFn?: () => PaymentMethod<GooglePayInitializationData>;
     private _isBuyNowFlow = false;
     private _currencyCode?: string;
+    private _currencyService?: CurrencyService;
```

Comment:
Where is this class instantiated?

Reply:
Hi @animesh1987 
this class instantiated here [https://github.com/bigcommerce/checkout-sdk-js/pull/2548/files#diff-344a206df74418e9cd10b2f25c10d6803c010e5b5b85133731267f1120321618R297](https://github.com/bigcommerce/checkout-sdk-js/pull/2548/files#diff-344a206df74418e9cd10b2f25c10d6803c010e5b5b85133731267f1120321618R297)